src/styles/Constants.scss

```diff
@@ -0,0 +1,2 @@
+$light-black: #000;
```

Comment:
I think this is the darkest black you can achieve. Why not naming it just `$black`?